tools/build-docs

```diff
@@ -1,3 +1,18 @@
-#!/usr/bin/env bash
-set -eu
-make -C "$(dirname "$0")"/../docs clean html
+#!/usr/bin/env python3
+
+# check for the venv
+from lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+import os
+import subprocess
+
+
+def main() -> None:
+    path = os.path.join(os.path.dirname(__file__), "../docs")
+    subprocess.call(["make", "clean", "html", "-C", path], stderr=subprocess.STDOUT)
```

Comment:
Rather than use this, can you use `run` from `scripts/lib/zulip_tools.py`?  That does a `check_call` and has various other minor benefits.

Reply:
Sure